apps/nextra/.env.example

```diff
@@ -1 +1,2 @@
-NEXT_PUBLIC_ORIGIN="http://localhost:3030"
\ No newline at end of file
+NEXT_PUBLIC_ORIGIN="http://localhost:3030"
+NEXT_PUBLIC_CODECACHE_API_KEY="<API_KEY>"
```

Comment:
I think this should be removed. If it's defined, it will try to make requests with this and they'll fail since its the wrong API_KEY.